jenkins-pipelines/qa/hydra-clean-test-resources.jenkinsfile

```diff
@@ -0,0 +1,49 @@
+#! groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+pipeline {
+    agent {
+        label {
+            label "aws-sct-builders-eu-west-1-v2-CI"
+        }
+    }
+    environment {
+        AWS_ACCESS_KEY_ID     = credentials('qa-aws-secret-key-id')
+        AWS_SECRET_ACCESS_KEY = credentials('qa-aws-secret-access-key')
+    }
+    parameters {
+        string(defaultValue: '',
+               description: 'ID of a SCT test to clean resources',
+               name: 'test_id')
+        string(defaultValue: 'aws',
+               description: 'resources backend',
+               name: 'backend')
+    }
```

Comment:
Need to add parameter for the region(s).
Then define `SCT_REGION_NAMES` env var with it's value.
Without it the SCT will pick up default value and if it doesn't match the test's region you will never get resources deleted using this job.

Reply:
ugh, in that case `hydra clean-resources` which is not accepting region as param does not work either?

Reply:
How about adding --regions parameter so it's clear that it must be provided and fallback to all regions if not?

Reply:
> How about adding --regions parameter so it's clear that it must be provided and fallback to all regions if not?

I don't mind.

Reply:
added and now it should work